packages/browser/src/authenticatedFetch/fetchFactory.ts

```diff
@@ -38,6 +38,7 @@ export function buildBearerFetch(
   return (init, options): Promise<Response> => {
     return fetch(init, {
       ...options,
+      credentials: "include",
```

Comment:
For my education: what's the difference between the `fetch` created by this factory, and the `fetchWithCookies` created in ClientAuthentication?

Reply:
`fetchFactory` creates an authenticated fetch from an access token, by setting the `Authorization` header. In this case, the `credentials: "include"` entry enables (if I understood it correctly) to set the cookies based on the response's `Set-Cookie` headers, which seem to be ignored otherwise. This authenticated fetch is the one that will allow the first access to a user resource and set the Resource Server cookie on the browser.

On the other hand, `fetchWithCookie` just enhances a "plain" fetch (so a non-authenticated one, without any `Authorization` header) so that if cookies exist on the browser, they get sent along the request. Note that the cookies need to have the `SameOrigin=None` attribute set, so that they can be sent on cross-orogin requests. Unlike the authenticated fetch, `fetchWithCookie` would never set a cookie if it doesn't exist: the request would simply return 401.